src/templates/infra/data/repository/postgres/repository.ejs

```diff
@@ -8,7 +8,9 @@ class <%- props.name.pascalCase %>Repository extends Repository {
         super({
             entity: <%- props.name.pascalCase %>,
             table: "<%- props.table %>",
-            knex: connection
+            ids: [<% for (const [index, value] of Object.entries(props.primaryKeyFields)) { %><%if(index != 0) { %>, <% } %>'<%= value %>'<% } %>],
```

Comment:
This part of the code seems duplicated in other files.

I think this should be refactored